backend/dataall/modules/datasets_base/db/dataset_repositories.py

```diff
@@ -53,9 +53,12 @@ def create_dataset(
         organization = Organization.get_organization_by_uri(
             session, environment.organizationUri
         )
+        suffix_prop = config.get_property('modules.datasets.features.dataset_suffix_tag', default=None)
```

Comment:
I think this was brought over from a different PR or from workshop - do not think it is needed here and may have been causing errors on my integration tests in quality gate